### PR TITLE
Alaa/feat/design

### PR DESCRIPTION
**Pull request by ashgolan**

### CODE REVIEW

File: src/components/advanced-search/AdvancedFields.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import * as S from "./AdvancedFields.styled";
+
+export default function AdvancedFields({ topMovies }) {
+	const years = topMovies
+		.map((movie) => {
+			return movie.year;
+		})
+		.sort()
+		.reverse();
+	// const genreArr = [];
```
Comment (AmeerLalaDeveloper) [inline src/components/advanced-search/AdvancedFields.jsx:11]: delete comments here

File: src/components/advanced-search/AdvancedFields.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+						<option key={`movie${index}`} value={year}>
+							{year}
+						</option>
+					);
+				})}
+			</S.Select>
+			{/* <select name='' id=''>
+					{genreArr &&
+						genreArr.map((genre, index) => {
+							return (
```
Comment (AmeerLalaDeveloper) [inline src/components/advanced-search/AdvancedFields.jsx:39]: delete comments here

File: src/layout/Header/Header.jsx [hunk 3]
```diff
@@ -1,11 +1,11 @@
 import Navbar from "./components/Navbar/Navbar";
-import SearchBar from "./components/SearchBar/SearchBar";
+// import SearchBar from "./components/SearchBar/SearchBar";
 import * as S from "./Header.styles";
 const Header = () => {
 	return (
 		<S.Container>
 			<Navbar />
-			<SearchBar />
+			{/* <SearchBar /> */}
```
Comment (AmeerLalaDeveloper) [inline src/layout/Header/Header.jsx:8]: delete comment here

Comment (ashgolan) [reply src/layout/Header/Header.jsx:8]: i put it into the navbar

**Pull request merged**